sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/directconnectivity/RntbdTransportClient.java

```diff
@@ -474,6 +474,8 @@ public static final class Options {
         @JsonProperty()
         private final Duration sslHandshakeTimeoutMinDuration;
 
+        @JsonProperty()
```

Comment:
should we add this to the Options builder example / java doc that we have?

Reply:
added javadoc